components/dashboard/src/projects/Prebuilds.tsx

```diff
@@ -159,7 +153,8 @@ export default function () {
                 <div className="py-3 pl-3">
                     <DropDown prefix="Prebuild Status: " contextMenuWidth="w-32" entries={statusFilterEntries()} />
                 </div>
-                <button disabled={!project} onClick={() => triggerPrebuild(null)} className="ml-2">Trigger Prebuild</button>
+                {(!!project && prebuilds.length === 0) &&
+                    <button onClick={() => triggerPrebuild(null)} className="ml-2">Run Prebuild</button>}
```

Comment:
**suggestion:** Also, although I think the run prebuild being always visible is the best approach long term (see https://github.com/gitpod-io/gitpod/issues/5374#issuecomment-905420796), instead of hiding this button and causing any confusion should we aim for addign a boring (simple) empty state like the one we have in workspaces for teams with no projects? 

| Workspaces list for teams with no projects | 
|-|
| <img width="1440" alt="Screenshot 2021-09-30 at 6 08 58 PM (2)" src="https://user-images.githubusercontent.com/120486/135482417-e90e98d8-efe9-478b-815f-ea92941ce0b9.png"> |

Reply:
Current state of the `Run Prebuild` button:
- It requests a new Prebuild for a specific branch (by default, the default branch, but we could make the button branch-specific, or ask with a modal)
- If there is already a successful Prebuild for the requested branch, the backend returns "a successful Prebuild already exists for this branch!" (but we don't show anything in the UI for that case)
- If there is a Prebuild for the requested branch, but it has failed, then the backend will actually trigger a new Prebuild to "try again" (this is new with this PR -- previously it would just say "there is already a (failed) Prebuild for this branch, doing nothing!")

Are you proposing to always allow triggering new Prebuilds? I.e.
- Even if there is a successful Prebuild for a branch (i.e. all the prebuilding work for the latest commit has already been done, and the result is ready to go) we force the creation of a new Prebuild for the same commit?